.ci.yaml

```diff
@@ -3231,7 +3231,18 @@ targets:
     properties:
       tags: >
         ["devicelab","ios32","mac"]
-      task_name: native_ui_tests_ios32
+      task_name: native_ui_tests_ios
```

Comment:
The target name `Mac_ios32 native_ui_tests_ios32` needs an update as well. Nit: also the tags: ios32->ios

Reply:
> Ownership check is failing:
> 
> ```
> # Test ownership check failed.
> Builders missing owner: [Mac_ios32 native_ui_tests_ios32]
> ```
> 
> However the test ownership is mapped by test file, and I already added a test owner for it. Does that mean test files cannot be shared between builders?

No problem to share test file, but IIRC the logic is based on `target` name, rather than the task_name in tags. /cc @chunhtai

Reply:
> No problem to share test file, but IIRC the logic is based on `target` name, rather than the task_name in tags. 

You were right, that was the problem.

> Nit: also the tags: ios32->ios

The old `ios32` tag wasn't correct?  Where was it running?